recipes/leftbar/unread_chip/unread_chip.vue

```diff
@@ -1,32 +1,27 @@
 <template>
-  <dt-chip
+  <button
     :class="['dt-leftbar-unread-chip', `dt-leftbar-unread-chip__${kind}`]"
```

Comment:
`__` is for a BEM child. Use `--`, which is a BEM variant as these are; e.g. `dt-leftbar-unread-chip--${kind}`